src/lang/Messages_bs_BA.properties

```diff
@@ -872,7 +872,7 @@ context.list.table.name=Kontekst
 context.list.table.inscope=U Opsegu
 context.name.prefix.label=Kontekst\:
 context.new.title=Novi kontekst
-context.prefixName=Context\: {0}
+context.prefixName=Komentar\: {0}
```

Comment:
Typo? Based on the above it should be `Kontekst`.

Reply:
Agreed

Reply:
Added a comment in Crowdin.